src/factory-in-proc.js

```diff
@@ -130,6 +130,7 @@ class FactoryInProc {
     }
 
     const node = new Node(options)
+    node.once('error', err => callback(err, node))
```

Comment:
If a node emits an errors *after* starting, won’t this result in `spawn`’s callback getting called more than once? (First with a success, then, later, with an error.)

Reply:
Shouldn't happen but ur right error should come first. Good catch, thank you